var/spack/repos/builtin/packages/py-py2cairo/package.py

```diff
@@ -33,9 +33,13 @@ class PyPy2cairo(Package):
 
     version('1.10.0', '20337132c4ab06c1146ad384d55372c5')
 
+    variant('X', default=True,
+            description='Enable cairo X11 dependency')
+
```

Comment:
Please remove this variant.
[Also... variants need to be lower-case name; although we haven't gotten around to fixing past transgressions yet.]